src/rust/catnip/runtime/mod.rs

```diff
@@ -131,41 +131,49 @@ pub struct SharedDPDKRuntime(SharedObject<DPDKRuntime>);
 /// Associate Functions for DPDK Runtime
 impl SharedDPDKRuntime {
     pub fn new(config: Config) -> Result<Self, Fail> {
+        let tcp_offload: Option<bool> = match config.tcp_checksum_offload() {
+            Ok(offload) => Some(offload),
+            Err(_) => None,
+        };
+
+        let udp_offload: Option<bool> = match config.udp_checksum_offload() {
+            Ok(offload) => Some(offload),
+            Err(_) => None,
+        };
+
         let (mm, port_id, link_addr) = Self::initialize_dpdk(
-            &config.eal_init_args(),
-            config.use_jumbo_frames(),
+            &config.eal_init_args()?,
+            config.enable_jumbo_frames()?,
             config.mtu()?,
-            config.tcp_checksum_offload(),
-            config.udp_checksum_offload(),
+            tcp_offload.unwrap_or(false),
+            udp_offload.unwrap_or(false),
         )
```

Comment:
Do not proceed silently. If options were not present warn about default behavior.

Reply:
done